### ISSUE TITLE
Validation Error handling in form builder is incomplete.

### ISSUE THREAD

**Issue by CharString** [open-formulieren/open-forms#3013]
### Product versie / Product version

2.1.x

### Omschrijf het probleem / Describe the bug

- when both FormVariable *and* ServiceFetchConfiguration validation errors occur when saving a form (the PUT to the bulk variable save) only the "Logic"-tab gets an error marker.
- errors in the ServiceFetchConfiguration are not marked in the modal

### Stappen om te reproduceren / Steps to reproduce

1. Create a User defined variable on the Variables tab. Set its type to "Time" and enter 25:00
2. Create a Logic trigger with an action "fetch the value for a variable from a service".
3. Configure the ServiceFetchConfiguration and make an error in it (i.e. add an empty row to the headers or make give an invalid Mapping expression.

### Verwacht gedrag / Expected behavior

This touches on the logging and the "try out" tab in terms of providing the form designer with the information they need in order to construct a complex form. Also [#2959](https://github.com/open-formulieren/open-forms/issues/2959) will give a new class of errors about logic expressions and variables: type mismatches

In short. In order to write complex forms, debugging information has to be communicated ergonomically.

### Screen resolution

None

### Device

None

### OS

None

**Comment by joeribekker** [comment]
Refinement: Business decision to not polish service fetch without budget from stakeholders. The feature remains experimental until its polished enough.
